package.json

```diff
@@ -45,6 +45,6 @@
     "mv": "2.1.1",
     "prettier": "2.0.4",
     "prettier-plugin-solidity": "1.0.0-alpha.48",
-    "solidity-coverage": "0.7.4"
+    "solidity-coverage": "0.7.5"
```

Comment:
can remove this change.

already added to Katalyst

Reply:
revert this change